app/webpacker/components/Live/components/ResultsTable.jsx

```diff
@@ -0,0 +1,172 @@
+import { Table } from 'semantic-ui-react';
+import React, { useMemo } from 'react';
+import _ from 'lodash';
+import { formatAttemptResult } from '../../../lib/wca-live/attempts';
+import { events } from '../../../lib/wca-data.js.erb';
+import { editRegistrationUrl, liveUrls } from '../../../lib/requests/routes.js.erb';
+
+const customOrderBy = (competitor, resultsByRegistrationId, sortBy) => {
+  const competitorResults = resultsByRegistrationId[competitor.id];
+
+  if (!competitorResults) {
+    return competitor.id;
+  }
+
+  return competitorResults[0][sortBy];
+};
+
+export const rankingCellStyle = (result) => {
+  if (!result) {
+    return {};
+  }
+  if (result.advancing) {
+    return { backgroundColor: 'rgb(0, 230, 118)' };
```

Comment:
The green tone should probably be stored somewhere, especially since you're reusing it with transparency again below